test/tests/cli.js

```diff
@@ -728,26 +643,26 @@ function initDat(opts, cb) {
     if (error || stdo.indexOf('Initialized dat store') === -1) {
       throw error || stdo.toString()
     }
-    
+
     // dont serve when in rpc mode
     if (opts.rpc) return done()
-    
-    var server = spawn(datCliPath, ['listen'], opts)
-    
+
+    var server = spawn('dat-server', ['listen'], opts)
```

Comment:
not sure what to do here. `dat-server` won't necessarily be available from the commandline -- maybe it makes sense to be able to start a server from the javascript without a Dat instance, and instead just a path to the dat. what do you think, @sethvincent ?